clojure-mode.el

```diff
@@ -191,38 +191,55 @@ Out-of-the box clojure-mode understands lein, boot and gradle."
           (and (listp value)
                (cl-every 'stringp value))))
 
+(defcustom clojure-mode-keymap-prefix "C-c C-r"
```

Comment:
Also, it should be called `clojure-mode-refactorings-keymap-prefix` or something like that. As only our refactoring commands are under this prefix.